.drone.yml

```diff
@@ -50,6 +80,9 @@ steps:
       registry: quay.io
       repo: quay.io/ukhomeofficedigital/hocs-search
       tags:
+        - build_${DRONE_BUILD_NUMBER}
+        - ${DRONE_COMMIT_SHA}
+        - branch_${DRONE_COMMIT_BRANCH/\//_}
```

Comment:
Nitpick but i'm not bothered about this one, it will always be branch_main

Reply:
Changed...